src/Bonnier/WP/Cxense/Assets/Scripts.php

```diff
@@ -105,19 +105,23 @@ public function head_tags()
     private function objects_to_array($items)
     {
         if (!is_wp_error($items) && !empty($items)) {
-
             // Only one? Just return it
-            if(count($items) === 1) {
-                return $items[0]->name;
+            if(is_array($items)){
+                if(count($items) === 1) {
```

Comment:
I also fixed this, since there was a ticket on it (GDS-273) - if the $items is an object, it will return true on count($items) === 1, and then it will fail when "return $items[0]->name"